airgun/views/template.py

```diff
@@ -0,0 +1,42 @@
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Text
+from widgetastic.widget import TextInput
+from widgetastic.widget import View
+
+from airgun.views.common import BaseLoggedInView
+from airgun.widgets import ActionsDropdown
+from airgun.widgets import RadioGroup
+
+
+class SyncTemplatesView(BaseLoggedInView):
+    title = Text("//h2[contains(., 'Import or Export Templates')]")
+    import_export = RadioGroup("//div[@class='col-md-6']")
+    submit = Text("//button[@type='submit']")
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(self.title, exception=False) and \
+               self.browser.wait_for_element(self.import_export) is not None
+
+    @View.nested
+    class ImportTemplates(View):
+        import_template = Text("//input[@value='import']")
+        associate = ActionsDropdown("//select[@name='associate']")
+        branch = TextInput("//input[@name='branch']")
```

Comment:
`TextInput` supports simplified specification, where you don't have to pass full location, but can use named argument `name` or `id`:
```python
branch = TextInput(name="branch")
```

The same applies to `Checkbox`. 

Up to, full locators are not wrong, strictly speaking. But simplified version is easier to read.